### DISCUSSION TITLE
Memory allocation error

### DISCUSSION THREAD

**Discussion by Akash47007** [micropython/micropython#15556] [RP2040 / Pico] [answered]
I doth understand what is happening here

![Screenshot 2024-07-26 171249](https://github.com/user-attachments/assets/7cd5fdeb-ae0d-4c0e-b401-f4bb47c99680)

![Screenshot 2024-07-26 171345](https://github.com/user-attachments/assets/9c9a4223-8bb8-4d53-96b8-5fc0ba0a09b9)

**Comment by GitHubsSilverBullet** [comment]
Allocating more than 0.5MB on a µC with about 260KB RAM usually leads to that kind of problem.

**Comment by scruss** [comment, marked as answer]
If you're trying to run `demo_animated_sprite.py` from the [micropython-ili9341](https://github.com/rdagger/micropython-ili9341) library, see the comment at the top of the file:
[Note:  This demo requires a board with additional PSRAM.](https://github.com/rdagger/micropython-ili9341/blob/dd749fb0c278112722a8a5f42e4fdd2644215b2b/demo_animated_sprite.py#L3)
The RP2040 does not support PSRAM

**Comment by Akash47007** [comment]
Is there any run around so that I can use it for my project
